### PR TITLE
Nav Bar UI Revamp

### PR DESCRIPTION
**Pull request by chaityabshah**
Worked on making Nav bar look similar to wireframe here: https://wiki.openmrs.org/display/projects/ID+Dashboard+Wireframes#IDDashboardWireframes-Dashboard-Home/ExplorePage
Slightly changed design to make it more elegant.

### CODE REVIEW

**Comment by elliottwilliams** [comment]
Nice work! It's a good start, but I noticed some bugs in the implementation. For instance, if there are too many links to show in a line, the nav bar wraps to the next line:

screen shot 2014-12-16 at 11 28 53

Perhaps you should put the nav-bar in a `.container` div, then use percent widths to keep from wrapping? You could also use some JS in the user-nav.ejs template to figure out how many links you had to show, and then setting up the [bootstrap grid widths](http://getbootstrap.com/css/#grid) to fit that many.

**Comment by elliottwilliams** [comment]
Also, could you send me a link to your Melange task?

**Comment by chaityabshah** [comment]
Worked on percent widths for a while, but could not get it to align the way I wanted it, so I switched the design over to bootstrap buttons. Thanks for the help! Any other suggestions?

**Comment by elliottwilliams** [comment]
Hey! I really like this, I think the bootstrap button-group is perfect for this. Two very small details though:
- do you think large-size buttons would look better? http://getbootstrap.com/components/#btn-groups-sizing
- could you the primary color (via `btn-primary` class)? I think the "OpenMRS Purple" would look better than the light blue here, since it fits in more with the rest of the site colors

Thanks!!

**Comment by chaityabshah** [comment]
Sounds good, made the changes. I like the purple better actually, thanks for the suggestion!
Heres the [Melange Task](https://www.google-melange.com/gci/task/view/google/gci2014/4730708136296448)

File: views/layouts/user-nav.ejs [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
-    <% var same = (name == list[i].viewName) ? true : false %>
-    <a<% if (same) { %> class="active"<% } %> href="<%= (same) ? "#" : list[i].url %>">
-      <li><% if (list[i].icon) { %><i class="<%= list[i].icon %>"></i><% } %><%= list[i].name %></li>
-    </a>
-  <% } %>
-</ul>
\ No newline at end of file
+<div class="container" style="padding: 0px">
+<div class="col-md-16" style="padding: 0px">
+  <div class="btn-group-lg btn-group-justified" role="group" style="margin:0px">
```
Comment (elliottwilliams) [inline views/layouts/user-nav.ejs]: I think this class actually need to be `btn-group btn-group-lg`. Right now, the btn-group styles aren't getting applied and the buttons look like they're having some weird border-radius problems :)

**Comment by chaityabshah** [comment]
[Here](http://gyazo.com/f4749a68c1becb41f1ad0f50122470fe) is what it looks like, I added an equal margin on both sides so that it would be centered. Let me know if there's a more efficient way to center the whole bar. Thanks :)

**Comment by elliottwilliams** [comment]
Awesome. Thanks so much!

**Comment by elliottwilliams** [comment]
> This task has been approved, but will not be merged into the repository until all GCI tickets in the ID Dashboard UI Revamp project have been submitted. We're doing this so that multiple students can submit their design implementation and we can select from the best collective implementation.

**Pull request closed without merging**